UnhollowerBaseLib/Attributes/Il2CppImplementsAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace UnhollowerBaseLib.Attributes
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public class Il2CppImplementsAttribute : Attribute
+    {
+        public Type[] Interfaces { get; private init; }
```

Comment:
```suggestion
        public Type[] Interfaces { get; }
```

Since this is an auto-initialized property, it doesn't need an `init`.